data/sql/http-archive/subdomains.sql

```diff
@@ -14,13 +14,13 @@ CREATE TEMPORARY FUNCTION
 OPTIONS
   ( library="gs://commonspeak-udf/URI.min.js" );
 SELECT
-  getSubdomain(origin) AS subdomain,
-  COUNT(origin) AS count
```

Comment:
Origin doesn't seem to work anymore. Using url it seems to be fixed.

Reply:
Thank you for the PR. Hope to work on this project some more in the coming weeks.